### PR TITLE
7756 - searchfield no results

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will fix no results text should not be selected.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/7756

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/searchfield/example-index.html
- After opening the above URL, search with any text which is not present in the search items
- When a search term is not present in the list `No results` text is shown, try to select the `No results text` as the search term value
- It should not be selected

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by jbrcna — CHANGES_REQUESTED** [review]
the fix is working. However, in Firefox when you click the **"No results"** text, the text becomes not visible.
image

**Review by glenlieorillo — CHANGES_REQUESTED** [review]
"No results" text becomes invisible upon hovering in Light and High Contrast Mode. (Firefox)

https://github.com/infor-design/enterprise/assets/102151352/6b8be275-96a5-4643-ae24-378191c8b665

**Comment by tjamesallen15** [comment]
You can now retest again @jbrcna @glenlieorillo Ty

**Review by glenlieorillo — DISMISSED** [review]
This is now working as expected. Thanks, @tjamesallen15!


https://github.com/infor-design/enterprise/assets/102151352/9af155c6-5e65-44cf-9aa5-999cc61f50db

**Review by jbrcna — APPROVED** [review]

File: src/components/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -924,6 +931,7 @@ Autocomplete.prototype = {
     this.closeList();
     this.element.focus();
 
+    // eslint-disable-next-line
```
Comment (tmcconechy) [inline src/components/autocomplete/autocomplete.js:934]: May not be needed?

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I see one issue. When you type xxx in the field and then click the no-search it acts as clearing the field. I dont think it should clear, should leave the text as is in the field

**Comment by tjamesallen15** [comment]
@tmcconechy i updated the PR by your request change.

**Review by tmcconechy — APPROVED** [review]
Thanks! Looks good!

**Review by glenlieorillo — COMMENTED** [review]
This is now working as expected.

https://github.com/infor-design/enterprise/assets/102151352/a50736aa-e7b0-4e5b-b101-a6130fa61bbf

**Pull request merged**